test/reactionsystem_structure/designating_parameter_types.jl

```diff
@@ -0,0 +1,106 @@
+### Fetch Packages and Set Global Variables ###
+
```

Comment:
I was planning to do more tests on this. However, MTK does not actually document non-default parameter types or what you can do with them. I tried asking the MTK people, but they kind of dodged the question. At least the system is in place now, and we can do stuff with it if we want to.